docs/contributing/index.md

```diff
@@ -1,5 +1,12 @@
 # Contributing
 
+## Contribution flow
+
+Did you find a software bug? -> Open an issue in the corresponding software repository
+Did you find strange behavior? -> [Ask in Q&A](https://github.com/autowarefoundation/autoware/discussions/categories/q-a)
+Do you have questions about Autoware functionality? -> [Ask in Q&A](https://github.com/autowarefoundation/autoware/discussions/categories/q-a)
+Do you want to add new features to Autoware? -> [Make a feature request](https://github.com/autowarefoundation/autoware/discussions/categories/feature-requests)
+
 This page explains how to contribute to Autoware.
 
 First, conform to the [Code of conduct](#contribution-workflows) section for the general manners in contributions.
```

Comment:
As you pointed out here, it was a mistake. :bow: 
https://github.com/autowarefoundation/autoware/discussions/202#discussioncomment-2601405

```suggestion
First, conform to the [Code of conduct](#code-of-conduct) section for the general manners in contributions.
```